storybook/stories/components/card/composite.js

```diff
@@ -55,21 +55,21 @@ export default function() {
                 <Box type="D">Box 3</Box>
               </Grid>
             </Grid>
-            <Grid size={5} dev={4} padding={[1]} align="top">
+            <Grid size={5} dev={4} padding={1} align="top">
               <Grid marginBottom={1}>
                 <h3>Subtitle here</h3>
               </Grid>
               <Grid>
                 <Grid dev={2}>
-                  <Grid dev={1} padding={[1, 0.5, 0, 0.5]}>
+                  <Grid dev={1} padding="1, 0.5, 0">
                     <Col size={6}>
                       <h3>Title</h3>
                     </Col>
                     <Col size={6} justify="right">
                       test
                     </Col>
                   </Grid>
-                  <Grid padding={[1, 0.5, 0, 0.5]}>
+                  <Grid padding="1, 0.5, 0">
```

Comment:
I updated a couple examples to demonstrate how these changes don't impact snapshot tests